testing/e2e.sh

```diff
@@ -315,18 +315,6 @@ function test_war_image_with_custom_run_flags() {
   EXPECT_CONTAINS "$(cat bazel-bin/testdata/war_image_with_custom_run_flags)" "-i --rm --network=host -e ABC=ABC"
 }
 
-function test_java_runfiles_image() {
-  cd "${ROOT}"
-  clear_docker
-  EXPECT_CONTAINS "$(bazel run "$@" testdata:java_runfiles_image)" "asdf"
```

Comment:
are we still testing this?

Reply:
its overkill, we have sufficient tests that run a simple java image and check its expected output, this is not adding any more value